src/SFA.DAS.ApprenticePortal.Web/Pages/Home.cshtml.cs

```diff
@@ -12,28 +11,25 @@ public class HomeModel : PageModel
     {
         private readonly IOuterApiClient _client;
         private readonly AuthenticatedUser _user;
-        private readonly NotificationAccessor _notifications;
 
         public string CourseName { get; set; } = null!;
         public bool ApprenticeshipComplete { get; set; } = false;
 
-        public HomeModel(IOuterApiClient client, AuthenticatedUser user, NotificationAccessor notifications)
+        public HomeModel(IOuterApiClient client, AuthenticatedUser user)
         {
             _client = client;
             _user = user;
-            _notifications = notifications;
         }
 
         public async Task OnGet()
         {
             try
             {
-                if (!_notifications.Notifications.Contains(HomeNotification.ApprenticeshipDidNotMatch))
-                    await PopulateProperties();
+                await PopulateProperties();
             }
             catch
             {
-                _notifications.Notifications.Add(HomeNotification.ApprenticeshipDidNotMatch);
+                // Safely ignore population errors

```

Comment:
I haven't had a chance to run this yet, so I don't know if this code is present elsewhere, but I was thinking, if the match failed there would be a redirect to the check your details page at this point.

Reply:
There is no link to the home page until after an apprentice has matched to an apprenticeship so I'm not sure it's necessary.

Also, this home page is going to be used to house other services as well as CMAD very shortly, some of those may not require a matched apprenticeship.

That was my thinking anyway.